spec/factories/groups.rb

```diff
@@ -2,8 +2,25 @@
 FactoryGirl.define do
   factory :group do
     size 1
-    status 'full'
-    association :leader, factory: :student_in_draw
+    association :leader, factory: :student_in_draw, intent: 'on_campus'
     draw { leader.draw }
```

Comment:
I just noticed (and this touches on our discussion earlier) - so if we override the leader with a student outside of any draw, then our factory will create an invalid group? Not a problem, per se, just an observation.